tools/viewer/main.rs

```diff
@@ -22,6 +23,10 @@ struct Cli {
     #[arg(short = 'I', name = "include path for other .slint files", number_of_values = 1, action)]
     include_paths: Vec<std::path::PathBuf>,
 
+    /// The first argument is the library name, and the second argument is the path to the library.
```

Comment:
There is no "first" and "second" argument.  It should just mention it should be in the format  name=path or something like that